app/components/qrcode.hbs

```diff
@@ -0,0 +1,10 @@
+
+<div class='qr-code-container'> 
+ <QrCode
+    data-test="qr-code"
+    @text= {{@text}}
+    @darkColor='#00000'
+    class = 'qr-code'
+  />
+ </div>
+ <p class='qr-code-text'>Check out our RDS mobile app</p>
```

Comment:
Is this used somewhere ? The code looks similar to `mobile.hbs` template

Reply:
removed